src/NuGet.Services.Validation.Orchestrator/Telemetry/TelemetryService.cs

```diff
@@ -35,8 +35,9 @@ public class TelemetryService : ITelemetryService, ISubscriptionProcessorTelemet
         private const string MessageHandlerDurationSeconds = OrchestratorPrefix + "MessageHandlerDurationSeconds";
         private const string MessageLockLost = OrchestratorPrefix + "MessageLockLost";
         private const string SymbolsMessageEnqueued = OrchestratorPrefix + "SymbolsMessageEnqueued";
-        private const string ExtractLicenseFileDuration = OrchestratorPrefix + "ExtractLicenseFileDuration";
+        private const string ExtractLicenseAndReadmeFileDuration = OrchestratorPrefix + "ExtractLicenseAndReadmeFileDuration";
```

Comment:
Could you add a suffix to clarify the unit here?

```suggestion
        private const string ExtractLicenseAndReadmeFileDurationSeconds = OrchestratorPrefix + "ExtractLicenseAndReadmeFileDurationSeconds";
```